actor.js

```diff
@@ -128,16 +139,16 @@ module.exports = class Kernel {
   }
 
   /**
-   * creates a new message
-   * @param {*} data
+   * creates an actor
+   * @param {Integer} type - the type id for the container
+   * @param {Object} message - an intial [message](https://github.com/primea/js-primea-message) to send newly created actor
    */
-  createMessage (opts) {
-    const message = new Message(opts)
-    this.ports.checkSendingPorts(message)
-    return message
+  createActor (type, message) {
+    const id = this._generateNextId()
+    return this.hypervisor.createActor(type, message, id)
   }
 
-  generateNextId () {
+  _generateNextId () {
```

Comment:
minor: could be cleaned up as
```
  _generateNextId () {
    return {
      nonce: this.state.nonce + 1,
      parent: this.id
    }
  }
```
? 

(...or `const id = ...` if you want to leave `id` declared for easier logging)